README.md

```diff
@@ -22,3 +22,12 @@ docker network create komodo_scale
 ```
 docker-compose up -d
 ```
+
+The seed node will launch and listen for connections on its P2P_PORT, once the ac image has connected to the miner image, the miner will start mining blocks.
```

Comment:
If possible, try to follow proper terminology. Image is built from Dockerfile and Container is started from image.
_once the ac image has connected to the miner image_ - change image->container